### PR TITLE
Some small cleanups in GasLiftWellState

### PR DESCRIPTION
**Pull request by akva2**

### CODE REVIEW

**Comment by akva2** [comment]
jenkins build this please

File: opm/simulators/wells/BlackoilWellModel.hpp [hunk 1]
```diff
@@ -109,7 +109,7 @@ namespace Opm {
             typedef typename BaseAuxiliaryModule<TypeTag>::NeighborSet NeighborSet;
             using GasLiftSingleWell = ::Opm::GasLiftSingleWell<TypeTag>;
             using GasLiftStage2 = ::Opm::GasLiftStage2<TypeTag>;
-            using GLiftWellState = ::Opm::GasLiftWellState<TypeTag>;
+            using GLiftWellState = GasLiftWellState;
```
Comment (joakim-hove) [inline opm/simulators/wells/BlackoilWellModel.hpp:112]: Or just use `GasLIftWellState`?

Comment (akva2) [reply opm/simulators/wells/BlackoilWellModel.hpp:112]: Sure.

**Comment by akva2** [comment]
jenkins build this please

**Review by bska — APPROVED** [review]
Thanks.  I'll merge into master.

**Pull request merged**